src/assets/scripts/InputController.js

```diff
@@ -467,17 +481,17 @@ export default class InputController {
 
             case KEY_CODES.LEFT_ARROW:
                 // shortKeys in use
-                if (window.gameController.game.option.get('controlMethod') === 'arrows') {
-                    this.$commandInput.val(`${currentCommandInputValue} \u2BA2`);
+                if (this._isArrowControlMethod()) {
+                    this.$commandInput.val(`${currentCommandInputValue}t l `);
```

Comment:
not high maintenance at all, I'm glad you called that out.  We were actually getting two spaces between callsign and first command, so I just fixed the glitch here.  Wrong idea.

I just looked through it and found where the callsign is added to the command line and removed the space there.  these commands will have the space on the left in my next commit.

Good eye!  never be shy about these kinds of things!!